components/brave_rewards/resources/extension/brave_rewards/background.ts

```diff
@@ -78,3 +78,47 @@ chrome.runtime.onConnect.addListener(function () {
     }
   })
 })
+
+const tipTwitterUser = (tweetMetaData: RewardsTip.TweetMetaData) => {
+  chrome.tabs.query({
+    active: true,
+    windowId: chrome.windows.WINDOW_ID_CURRENT
```

Comment:
just to double check. Is it possible for `WINDOW_ID_CURRENT` wrong. Should we use `chrome.windows.getCurrent(callback)`?

Reply:
I think it's pretty safe to rely on `WINDOW_ID_CURRENT`: https://developers.chrome.com/extensions/windows#property-WINDOW_ID_CURRENT